package/src/components/ImageGallery/components/ImageGalleryVideoControl.tsx

```diff
@@ -74,18 +82,20 @@ export const ImageGalleryVideoControl = React.memo(
         </TouchableOpacity>
         <Text
           accessibilityLabel='Progress Duration'
-          style={[styles.durationTextStyle, { color: black }, durationTextStyle]}
+          style={[styles.progressDurationText, { color: black }, progressDurationText]}
```

Comment:
Technically a breaking change, but would provide more extensibility

Reply:
I would go with it. Else we can never improve this kinda things. Unless you have something to say.